www/css/cytube.css

```diff
@@ -266,16 +238,24 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 
 .server-msg-disconnect {
     border: 1px solid #ff0000;
-    margin: 5px;
-    padding: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
     color: #ff0000;
+    text-align: center;
+    background-color: rgba(129, 20, 21, 0.1);
 }
 
 .server-msg-reconnect {
     border: 1px solid #009900;
-    margin: 5px;
-    padding: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
     color: #009900;
+    text-align: center;
```

Comment:
I think it looks quite neat across all themes, if you think it looks bad or have a problem with it I can make it theme specific though. Is there anything specific you don't like about it?